Tests/Pcap++Test/Tests/TcpReassemblyTests.cpp

```diff
@@ -55,6 +55,28 @@ struct TcpReassemblyMultipleConnStats
 
 };
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// TcpReassemblyMultipleConnStats
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+struct TcpReassemblyMultipleConnStatsWithReassembly
+{
+	typedef std::vector<uint32_t> FlowKeysList;
+	typedef std::map<uint32_t, TcpReassemblyStats> Stats;
+
+	Stats stats;
+	FlowKeysList flowKeysList;
+
+	std::vector<timeval>timestamps;
+	void clear()
+	{
+		stats.clear();
+		flowKeysList.clear();
+	}
+
+	pcpp::TcpReassembly *tcpReassmbly = nullptr;
```

Comment:
Maybe instead of duplicating the entire struct we can just inherit from `TcpReassemblyMultipleConnStats` and add `pcpp::TcpReassembly *tcpReassmbly`?
Or another option is to add `pcpp::TcpReassembly *tcpReassmbly` to `TcpReassemblyMultipleConnStats`